db/models.py

```diff
@@ -1 +1,42 @@
 from django.db import models
+
+
+# Race model
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+# Skill model
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
```

Comment:
The `bonus` field in the `Skill` model is defined as a `CharField`. If this field is intended to store numerical values, consider using an `IntegerField` or `FloatField` instead. This will ensure that the data type is appropriate for numerical operations and validations.